tests/tests/ota

```diff
@@ -0,0 +1,34 @@
+#!/bin/env bash
+
+utils_dir="$(readlink -f $0 | xargs dirname)/../utils"
+
+ver_before=${utils_dir}/get_ver
+v="$((RANDOM % 128)).$((RANDOM % 128)).$((RANDOM % 128))"
+echo "building version ${v}"
+${utils_dir}/set_src_ver ${v}
+
+cd ${PRJDIR}/${SAMPLE_PATH}
+./build.sh ${BOARD}
+cd - &>> /dev/null
+
+echo "building done"
+
+d=$(dirname $(find ${PRJDIR}/${SAMPLE_PATH} -name 'zephyr.signed.bin'))
+f=$(find ${PRJDIR}/${SAMPLE_PATH} -name 'zephyr.signed.bin')
+
+echo "start server ${MY_IP}:${OTA_PORT}"
+python3 -m http.server --bind ${MY_IP} --directory ${d} ${OTA_PORT} &
```

Comment:
Can we use ```command & pid=$! ``` (or any other way) for start command and get PID for kill process?

Reply:
ok, I give up :) will use pid :)

but if server decide to fork itself we keeping child processes, and kill only parent one